layouts/css/_variables.styl

```diff
@@ -2,9 +2,10 @@ $body-max-width = 980px
 
 $white = #fff
 
-$node-green = #80bd01
-$light-green = #f1fbda
-$hover-green = #d9ebb3
+$node-green = #43853d
```

Comment:
Why not `#44883e` (http://rgb.to/pantone/7741-c)? I'd say it's good enough contrast.